ersilia/core/tracking.py

```diff
@@ -81,53 +85,73 @@ def log_files_metrics(file):
                 if misc_error_flag:
                     errors[error_name] += 1
 
-        write_persistent_file(f"Error count: {error_count}")
+        write_persistent_file(f"Error count: {error_count}", model_id)
         if len(errors) > 0:
-            write_persistent_file(f"Breakdown by error types:")
+            write_persistent_file(f"Breakdown by error types:", model_id)
             for error in errors:
-                write_persistent_file(f"{error}: {errors[error]}")
-        write_persistent_file(f"Warning count: {warning_count}")
+                write_persistent_file(f"{error}: {errors[error]}", model_id)
+        write_persistent_file(f"Warning count: {warning_count}", model_id)
     except (IsADirectoryError, FileNotFoundError):
         logging.warning("Unable to calculate metrics for log file: log file not found")
-        
+
+
+
+def persistent_file(model_id):
+
+    """
+    Create the persistent path file 
+    :param model_id: Te currently running model
+    """
+
+    persistent_file_dir = os.path.join(EOS, ERSILIA_RUNS_FOLDER, "session", model_id)
+    os.makedirs(persistent_file_dir, exist_ok=True)  
+    file_name = os.path.join(persistent_file_dir, "current_session.txt")
+    
+    return file_name
+
+    
 
 def open_persistent_file(model_id):
```

Comment:
You can get rid of this function entirely and move the print statement here to `create_persistent_file`